quant.module

```diff
@@ -180,12 +181,14 @@ function quant_node_access(NodeInterface $node, $op, AccountInterface $account)
   try {
     \Drupal::service('quant.token_manager')->validate($url, TRUE);
   }
+  catch (TokenValidationDisabledException $e) {
+    \Drupal::logger('quant')->info('Token validation is disabled, consider enabling');
```

Comment:
This would be noisy in the logs, esp during initial seed of 10s of thousands of items..